test/functional/Completion.hs

```diff
@@ -18,10 +19,13 @@ getResolvedCompletions :: TextDocumentIdentifier -> Position -> Session [Complet
 getResolvedCompletions doc pos = do
```

Comment:
don't you have a test helper for this? I guess it's in `lsp` and we need to do a release

Reply:
Yes it's part of the lsp-test pr I did.